smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/auth/AuthUtils.java

```diff
@@ -82,6 +82,13 @@ public static Map<ShapeId, HttpAuthScheme> getAllEffectiveNoAuthAwareAuthSchemes
                 effectiveAuthSchemes.put(shapeId, authIndex.getHttpAuthScheme(shapeId));
             }
         }
+        // TODO(experimentalIdentityAndAuth): remove after @aws.auth#sigv4a is fully supported
```

Comment:
can you mark the start and end of this TODO? otherwise it's not clear what lines are being referenced